tests/Feature/SnmpTraps/VeeamTrapTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace LibreNMS\Tests\Feature\SnmpTraps;
+
+use App\Models\Device;
+use LibreNMS\Snmptrap\Dispatcher;
+use LibreNMS\Snmptrap\Trap;
+use Log;
+
+class VeeamTrapTest extends SnmpTrapTestCase
+{
+    public function testVeeamOnBackupCompleted(): void
+    {
+        $device = Device::factory()->create();
+
+        $trapText = "$device->hostname
+UDP: [$device->ip]:46024->[1.1.1.1]:162
+DISMAN-EVENT-MIB::sysUpTimeInstance 4:13:08:37.60
+SNMPv2-MIB::snmpTrapOID.0 VEEAM-MIB::onBackupCompleted
+VEEAM-MIB::backupJobId 7a1b3549-c4c7-4629-84d6-74e24fee8011
+VEEAM-MIB::backupJobName SureBackup Job
+VEEAM-MIB::sourceHostName hostname
+VEEAM-MIB::vmBackupComment 
+VEEAM-MIB::vmBackupResult Success";
+
+        $trap = new Trap($trapText);
+
+        $message = 'SNMP Trap: Backup success - SureBackup Job 1';
+        Log::shouldReceive('event')->once()->with($message, $device->device_id, 'backup', 2);
+
```

Comment:
Looks like you are getting an extra log entry the tests don't expect or your log message doesn't match.

Reply:
mmh, I do not find my issue for a view days. No idea how to debug this test cases :-( Can someone help me?